### PR TITLE
Be queries2

### LINKED ISSUE #47: BE Users Query
Update User's query to include

- [x] Description
- [x] Address 
- [x] Image
- [x] Products
- [x] UserProducts

### PR DESCRIPTION
**Pull request by catherinemdean15**
### Description
Complete user by id query to include both products and userProducts (nested in products)
Update models and types to allow functionality

#### Closes Issues 
47 

## Proposed Changes
Update models and types to allow functionality

### CODE REVIEW

File: code/api/src/modules/userProducts/types.js [hunk 1]
```diff
@@ -4,17 +4,17 @@ import UserType from '../user/types'
...
+  description: 'User Product Type',
 
   fields: () => ({
     id: { type: GraphQLInt },
-    userId: { type: UserType },
-    productId: { type: ProductType },
-    delivered_date: { type: GraphQLString },
+    userId: { type: GraphQLInt },
+    productId: { type: GraphQLInt },
+    deliveryDate: { type: GraphQLString },
```
Comment (gaj23) [inline code/api/src/modules/userProducts/types.js:15]: Thank you for you persistence in this manner.

**Review by gaj23 — APPROVED** [review]
This is exciting! I'm looking forward to talking about how you over came this huge blocker that  yinz been struggling with.

File: code/api/src/modules/product/resolvers.js [hunk 2]
```diff
@@ -21,8 +21,11 @@ export async function getBySlug(parentValue, { slug }) {
 
 // Get product by ID
 export async function getById(parentValue, { productId }) {
-  const product = await models.Product.findOne({ where: { id: productId } })
-
+  const product = await models.Product.findOne({
+    where: { id: productId },
+    include: [{model: models.UserProduct }],
```
Comment (srslie) [inline code/api/src/modules/product/resolvers.js:26]: That's so great you guys got this hooked up! Thank you!

File: code/api/src/modules/userProducts/types.js [hunk 3]
```diff
@@ -4,17 +4,17 @@ import UserType from '../user/types'
 import ProductType from '../product/types'
```
Comment (srslie) [inline code/api/src/modules/userProducts/types.js:5]: Seems touchy about camelcasing and thank you for your diligence on pluralization.

**Review by srslie — APPROVED** [review]
Thanks for your work figuring that out!

**Review by tashiad — APPROVED** [review]
Thanks for your hard work on this!

**Pull request merged**